### PR TITLE
Update dependency management

### PR DESCRIPTION
**Pull request by bonniegee**
- Add dependabot config
- Add requirements.in file
- Update pre-commit config

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+coverage==5.5
```
Comment (p-galligan) [inline requirements.in:1]: You probably don't need coverage, right? Is it being used in the app somewhere?

**Review by p-galligan — CHANGES_REQUESTED** [review]

**Review by p-galligan — APPROVED** [review]
Looks good.

I noticed you're using coverage in the travis file, so I think it makes sense to include it.

**Comment by helrond** [comment]
Hmm yeah I'm not so sure about keeping `coverage` in the requirements file. It's true that the tests are being called via coverage but nothing is being done with the report. Even if that's changed though, I'd lobby for installing `coverage` in travis (as is done with `pre-commit` rather than declaring it as an app dependency.

**Comment by helrond** [comment]
Need to add `coverage` to the `pip install` step in Travis :)

**Review by helrond — APPROVED** [review]

File: requirements.in [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Django==3.2.5
+django-auth-adfs==1.6.0
+django-crispy-forms==1.12.0
+djangorestframework==3.12.4
+health-check==3.4.1
+psycopg2-binary==2.9.1
+python-dateutil==2.8.1
+rac_schemas==0.26
+requests==2.25.1
```
Comment (HaSistrunk) [inline requirements.in:9]: Do you need `requests` as a top-level dependency? See this discussion in the pisces dependency PR: https://github.com/RockefellerArchiveCenter/pisces/pull/447#issuecomment-878294274

Comment (bonniegee) [reply requirements.in:9]: Yep--I use it directly in a file.

**Pull request merged**